packages/swapper/src/swappers/thorchain/ThorchainSwapper.ts

```diff
@@ -31,11 +31,21 @@ import { thorService } from './utils/thorService'
 
 export class ThorchainSwapper implements Swapper<ChainId> {
   readonly name = 'Thorchain'
-  private swapSupportedChainIds: Record<ChainId, boolean> = {
+  private sellSupportedChainIds: Record<ChainId, boolean> = {
     [KnownChainIds.EthereumMainnet]: true,
     [KnownChainIds.BitcoinMainnet]: true
   }
-  private supportedAssetIds: AssetId[] = []
+
+  private buySupportedChainIds: Record<ChainId, boolean> = {
```

Comment:
is there a reason these are maps vs just arrays?

Reply:
used as a lookup